Directory.Build.props

```diff
@@ -0,0 +1,15 @@
+<Project>
+
+  <Sdk Name="DotNet.ReproducibleBuilds.Isolated" Version="1.1.1" />
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
```

Comment:
Please leave TFMs in individual projects - there are subtleties to project files that various tooling (not just Ionide, but VS/Rider occasionally as well) get hung up on when TFM isn't in the 'raw' project.

Reply:
I see. I have never experienced this in Rider. Could you give me some more information, or links where I can read more?

I'll have this fixed shortly.